packages/serverless-orchestration/package.json

```diff
@@ -5,7 +5,9 @@
   "dependencies": {
     "@truffle/contract": "^4.2.20",
     "@uma/common": "^2.0.1",
-    "@uma/financial-templates-lib": "^2.0.1"
+    "@uma/financial-templates-lib": "^2.0.1",
+    "truffle": "^5.2.3",
+    "web3": "^1.3.4"
```

Comment:
Is there a `require("truffle")` or `require("web3")` in any non-tests in this package? If not, I think we could probably drop these and rely on the top-level dev dependencies.

Reply:
I think there is a `require("web3")` in this package, but we can probably drop truffle.

Reply:
I dropped truffle, kept web3